client/ayon_core/hosts/maya/api/lib.py

```diff
@@ -2168,12 +2162,20 @@ def get_related_sets(node):
     sets = [s for s in sets if s not in deformer_sets]
 
     # Ignore when the set has a specific suffix
-    sets = [s for s in sets if not any(s.endswith(x) for x in ignore_suffices)]
+    ignore_suffices = ("out_SET", "controls_SET", "_INST", "_CON")
```

Comment:
There are 3 list comprehensions, could that be simplified to:
```python
    # Ignore when the set has a specific suffix
    ignore_suffices = ("out_SET", "controls_SET", "_INST", "_CON")
    # Default nodes to ignore
    defaults = {"defaultLightSet", "defaultObjectSet"}
    sets = [
        s
        for s in sets
        if (
            s not in deformer_sets
            and s not in defaults
            not s.endswith(ignore_suffices)
        )
]
```

Reply:
Or you can use tuple in `endswith`?

Reply:
I could combine them - sure. 

Note that `sets = [s for s in sets if not any(s.endswith(x) for x in ignore_suffices)]` can also be written as ` sets = [s for s in sets if not s.endswith(ignore_suffices)]`. `str.endswith` can actually take a `tuple`. 🤯

Reply:
Cool, modified the snippet.

Reply:
Changed the code with https://github.com/ynput/ayon-core/pull/324/commits/f3324c3863598f1946525aef02cbd8abd5444637